### DISCUSSION TITLE
my fine tuned whisper model does not have pytorch_model.bin

### DISCUSSION THREAD

**Discussion by Atefeh197** [openai/whisper#2162] [General]
I fine-tuned a whisper model by this link 
https://colab.research.google.com/github/sanchit-gandhi/notebooks/blob/main/fine_tune_whisper.ipynb#scrollTo=e0ace3aa-1ef3-45cb-933f-6ddca037c5aa

but there is no pytorch_model.bin  in the saved model folder,

**Comment by itaipee** [comment]
which files did you get ? something like "model-00001-of-00002.safetensors"  ? 

you can use this command line function to convert it 
 ct2-transformers-converter --model 'checkpoint directory'  --output_dir 'new directory' --copy_files preprocessor_config.json
